README.md

```diff
@@ -32,34 +57,41 @@ To get a local copy up and running follow these simple example steps.
 
 ## Authors
 
-👤 **Author1**
+👤 **habtamu birru**
 
 - GitHub: [@Habtamu210](https://github.com/Habtamu210)
 - Twitter: [@beleyaha](https://twitter.com/beleyaha)
 - LinkedIn: [Habtamu Birru](https://linkedin.com/in/habtamu-birru-4187ab20/)
 
-👤 **Author2**
-
-- GitHub: [@](https://github.com/)
-- Twitter: [@](https://twitter.com/)
-- LinkedIn: [](https://www.linkedin.com/in/)
+👤 **Ibute Uchenna**
 
+- GitHub: [@Uchennaibute](https://github.com/Uchennaibute@github)
+- Twitter: [@Uc_Ibute](https://twitter.com/Uchennaibute@twitter)
+- LinkedIn: [Uchenna Ibute](https://linkedin.com/in/Uchennaibute@linkedin)
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Habtamu210/Awesome-books/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
 
-## Show your support
 
-Give a ⭐️ if you like this project!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Acknowledgments
+## ❓ FAQ <a name="faq"></a>
 
-- 
-- 
-- 
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] @team as previously requested kindly ensure the link to your MIT license, it doesn't work since your file is MIT.md and not LICENSE.md.